features/org.wso2.carbon.siddhi.extensions.installer.core.feature/src/main/resources/extensionsInstaller/extensionDependencies.json

```diff
@@ -609,6 +784,126 @@
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])store([_-])rdbms([_-])6.0.4.jar"
+      },
+      {
+        "name": "mssql-jdbc",
+        "version": "8.2.1.jre8",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/com/microsoft/sqlserver/mssql-jdbc/8.2.1.jre8/mssql-jdbc-8.2.1.jre8.jar"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "mssql([_-])jdbc([_-])8.2.1.jre8(.+).jar"
+      }
+    ]
+  },
+
+  "rdbms-postgresql": {
+    "extension": {
+      "name": "rdbms-postgresql",
+      "displayName": "RDBMS - Postgre SQL",
+      "version": "6.0.4"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-store-rdbms",
+        "version": "6.0.4",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/store/rdbms/siddhi-store-rdbms/6.0.4/siddhi-store-rdbms-6.0.4.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])store([_-])rdbms([_-])6.0.4.jar"
+      },
+      {
+        "name": "postgresql",
+        "version": "42.2.10",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.10/postgresql-42.2.10.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "postgresql([_-])42.2.10(.+).jar"
+      }
+    ]
+  },
+
+  "rdbms-oracle": {
+    "extension": {
+      "name": "rdbms-oracle",
+      "displayName": "RDBMS - Oracle",
+      "version": "6.0.4"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-store-rdbms",
+        "version": "6.0.4",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/store/rdbms/siddhi-store-rdbms/6.0.4/siddhi-store-rdbms-6.0.4.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])store([_-])rdbms([_-])6.0.4.jar"
+      },
+      {
+        "name": "ojdbc8",
+        "version": "12.2.0.1",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://maven.xwiki.org/externals/com/oracle/jdbc/ojdbc8/12.2.0.1/ojdbc8-12.2.0.1.jar"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "ojdbc8([_-])12.2.0.1(.+).jar"
```

Comment:
Won't there be `(.*)` instead of `(.+)` at the end, similar to the other ones we have changed above?